android/src/main/kotlin/com/braze/brazeplugin/BrazePlugin.kt

```diff
@@ -162,7 +162,18 @@ class BrazePlugin : MethodCallHandler, FlutterPlugin, ActivityAware {
             when (call.method) {
                 "changeUser" -> {
                     val userId = call.argument<String>("userId")
-                    Braze.getInstance(context).changeUser(userId)
+                    val sdkAuthSignature = call.argument<String>("sdkAuthSignature")
+                    if (sdkAuthSignature != null) {
+                        Braze.getInstance(context).changeUser(userId, sdkAuthSignature)
+                    } else {
+                        Braze.getInstance(context).changeUser(userId)
```

Comment:
changeUser(userId) will just turn around and call changeUser(userId, null), so we could simplify this a little bit.

Reply:
@bryanlogan updated according to the comment.